Sources/MenuAnimator.swift

```diff
@@ -385,13 +432,19 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
                 }
                 
             case .changed:
+                guard let transitionContext = transitionContext else {
```

Comment:
No, in this situation we get a crash in the case when a user uses interactive transition before button clicking. So 'transitionContext' is not initialized before the switch.